.github/styles/Google/WordList.yml

```diff
@@ -75,5 +81,7 @@ swap:
   tablet: device
   touch: tap
```

Comment:
in that case should tap & hold remain as is and not touch & hold?

Reply:
Double-checked Google's style guide and they prefer 'touch & hold' (with the ampersand). I can edit this so it doesn't catch "touch & hold" (...but do we ever use the term? we don't have any mobile-first docs, do we?)

Reply:
no, this doesn't apply to us.